docs/source/troubleshooting.rst

```diff
@@ -74,11 +70,12 @@ Chrome <https://developer.chrome.com/devtools/docs/cpu-profiling>`_ to find
 bottlenecks in the code.
 
 However, with large rosters (more than 1000 contacts), rendering in
-Converse slows down a lot and it may become intolerably slow.
+Converse is known to slow down considerbly. It may become intolerably slow
```

Comment:
Typo: considerably

Reply:
@jcbrand Thanks, fixed a few more typos. Thanks for all your work, this is a terrific XMPP client.